app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -0,0 +1,148 @@
+// govuk_frontend_toolkit
+@import 'grid_layout';
+@import 'design-patterns/buttons';
+
+.pub-c-feedback {
+  max-width: 960px;
+  margin: 0 auto;
+}
+
+// hide without js
+// show with js, unless also has the js-hidden class
+.pub-c-feedback__js-show {
+  display: none;
+
+  .js-enabled & {
+    display: block;
+
+    &.js-hidden {
+      display: none;
+    }
+  }
+}
+
+.pub-c-feedback__grid-row {
```

Comment:
Is there a reason why we're not able to just use the `grid-row` and `column-two-thirds` classes directly on the HTML elements?

Reply:
Yes. I don't think they're included in all of our frontend apps - I initially tried this component in feedback (I think it was) and it didn't have the grid.

Reply:
That makes sense, thanks for explaining! I had assumed importing the grid_layout scss here would mean it was available whenever the component is used, but I just remembered these components work a bit differently.

Reply:
Also I guess there's an ongoing question over the isolation of these components and how much we should assume is available to them. But that's a story for another day. Happy to approve?